src/user/controller/user.controller.ts

```diff
@@ -143,6 +146,12 @@ export class UserController {
     return this.userService.updateSettings(user.id, data);
   }
 
+  @Patch('onboard')
+  @BypassOnboardingCheck()
+  async onboardUser(@CurrentUser() user: User) {
+    return this.userService.onboardUser(user);
```

Comment:
This should return an user with the properties that we're usually sending with the user (those from `select: ...`).

Reply:
Hmmmmmmm gotcha